metadata.tvshows.thetvdb.com.v4.python/resources/lib/artwork.py

```diff
@@ -15,9 +15,13 @@ def add_artworks(show, liz, language):
     for art_type, images in artworks.items():
         for image in images[:MAX_IMAGES_NUMBER]:
             liz.addAvailableArtwork(image['image'], art_type)
-
+    
+    season_posters_dict = defaultdict(list)
     for image, season_number in season_posters:
-        liz.addAvailableArtwork(image, 'poster', season=season_number)
+        season_posters_dict.setdefault(season_number, []).append(image)
```

Comment:
No need to use setdefault for defaultdict(list). It already provides empty lists on the first access by key.